pkg/clients/clients.go

```diff
@@ -55,6 +56,7 @@ type ClientsInterface interface {
 	CreateOrUpdate(ctx context.Context, obj client.Object, fn controllerutil.MutateFn) (controllerutil.OperationResult, error)
 	HasResource(resource schema.GroupVersionResource) (bool, error)
 	GetPlatform() (string, error)
+	GetNodesByLabels(ctx context.Context, matchingLabels map[string]string) (*v1.NodeList, error)
```

Comment:
Shall we keep the alphabetical order here? This line should be just above `GetPlatform`.

Reply:
Fixed